ev3dev/helper.py

```diff
@@ -311,28 +311,56 @@ def __init__(self, motor_specs, desc=None):
         (self.left_motor, self.right_motor) = self.motors.values()
         self.max_speed = self.left_motor.max_speed
 
-    def set_speed_ratio(self, left_motor_ratio, right_motor_ratio, set_right=True):
+    def set_speed_steering(self, direction, power=100):
         """
-        Set the speeds of the left_motor vs right_motor by ratio
+        Set the speed_sp for each motor in a pair to achieve the specified steering
+
+        direction [-100, 100]:
+            * -100 means turn left as fast as possible,
+            *  0   means drive in a straight line, and
+            *  100 means turn right as fast as possible.
+
+        power: the power that should be applied to the outmost motor (the one
```

Comment:
_power_? I think this should probably be "speed"

Reply:
Caught that in the initial cut-n-paste....will change all of the power references to speed